docker/spark-on-k8s/README.md

```diff
@@ -0,0 +1,44 @@
+--------
+# Dockfiles for spark-on-k8s
+
+## Before the build
+
+Please download the spark-2.2.0-k8s distribution like spark-2.2.0-k8s-0.5.0-bin-with-hadoop-2.7.3.tgz and unzip it. Replace the dockerfiles with dockerfiles here.
+
+## How to build it
+
+### First, build the latest images for spark-base, spark-driver, spark-driver-py, spark-executor, spark-executor-py and spark-init.
+
+    sudo docker build -t spark-base:latest \
```

Comment:
Should we set the meaningful tag here directly using the -t option? Like `sudo docker build -t intelanalytics/spark-base:v2.2.0-kubernetes-0.5.0-ubuntu-14.04`

Reply:
no, because the other dockerfiles are "from spark-base" which means spark-base should be latest at the first place, otherwise we have to modify all the dockerfiles to from intelanalytics/spark-base:v2.2.0-kubernetes-0.5.0-ubuntu-14.04